src/mango/test/05-index-selection-test.py

```diff
@@ -201,8 +203,11 @@ def test_manual_bad_view_idx01(self):
                 }
             },
         }
-        with self.assertRaises(KeyError):
+        try:
             self.db.save_doc(design_doc)
+            raise("should not get here")
```

Comment:
This isn't valid Python:

```python
>>> raise("foo")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: exceptions must be old-style classes or derived from BaseException, not str
```

The Python-ism you want is:

```python
assert False, "Should not get here."
```

Reply:
fixed